website/docs/concepts/html/elements.md

```diff
@@ -19,6 +19,43 @@ html! {
 }
 ```
 
+## Boolean Attributes 
+
+Some content attributes (e.g checked, hidden, required) are called boolean attributes. In Yew, 
+boolean attributes need to be explicitly set to 'true':
+
+```rust
+    html! {
+        <div hidden=true>
+            { "This div is hidden." }
+        </div>
+    }
+```
+
+This will result in the following **HTML**: 
+```html
+    <div hidden>This div is hidden.</div>
+```
```

Comment:
This isn't exactly how Yew emits boolean attributes (blame the HTML spec) but this is much easier to understand than the real thing.
Perhaps we could clarify that the actual output would be equivalent to this so it's technically correct?

 ```suggestion
This will result in **HTML** that's functionally equivalent to this: 
```html
    <div hidden>This div is hidden.</div>
```
```

Reply:
Ohh I see, Yew emits attribute='attribute' from the spec :)